### DISCUSSION TITLE
Multivariant problem and Problem with Heroku

### DISCUSSION THREAD

**Discussion by ayon120** [saleor/saleor#8344] [General]
Hello,

I am new to Saleor, I would be very grateful if anybody could assist me on the following 2 things.

**1.** I have created a custom product type with 2 variant types(size & color). I want my product to have 2 or more varients but If I add more than 1 variant the options look like the picture below. (multiple options of a single attribute are being created for example: size and color as shown in the picture) Am I doing this wrong or is there a workaround to this?
![Capture](https://user-images.githubusercontent.com/30633846/136930704-172185d4-fc53-4cb2-bd62-207108e7e7c9.JPG)
![Capture2](https://user-images.githubusercontent.com/30633846/136930758-d0539035-5a4c-4167-a509-92e9a8812aec.JPG)

**2.** I have uploaded the saleor storefront and saleor core in Heroku. My problem is that, If I add a product to the cart and try to buy it with the built-in dummy payment plugin here, I get an error and get stuck on the page. I can not go to the review page to finally confirm the product purchase. But it was fine when I was doing this in the local environment. I have pasted a screenshot.
![Capture3](https://user-images.githubusercontent.com/30633846/136933556-6fd86785-d103-45e6-8787-45bfbe10db3d.JPG)

**Comment by mbeprint** [comment]
Hi,
Did you find a solution for this issue? I have come across as well and asked in discussion #8000 . They are aware of the issue but I have not seen any update.

**Reply by karolkielecki** [reply]
Hi! Thanks for reporting. Yes, we're aware of the issue but haven't fixed it yet. If I have any news, I'll let you know.
